doc/AWS.md

```diff
@@ -0,0 +1,466 @@
+
+**Encryption**
+
+Check **Require HTTPS for all traffic to the domain**. Do not select **Node-to-node encryption** or **Enable encryption of data at rest**. There is no need to fill in any of the **Optional Elasticsearch cluster settings**.
+
+Now click `Confirm` to create your ElasticSearch instance. 
+
+To aid development you may want to access ElasticSearch and Kabana from your development machine. If this is the case then select your domain and in the `Actions` drop down select **Modify access policy)**. Ensure JSON defined access policy is set and paste the following JSON:
+
+```
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": "USER_ARN"
+      },
+      "Action": "es:*",
+      "Resource": "ELASTIC_SEARCH_ARN"
+    },
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": "*"
+      },
+      "Action": "es:ESHttp*",
+      "Resource": "ELASTIC_SEARCH_ARN",
+      "Condition": {
+        "IpAddress": {
+          "aws:SourceIp": "DEV_SERVER_IP"
+        }
+      }
+    }
+  ]
+}
+```
+
+Making the following changes:
+ * **USER_ARN**: You should find this on the summary screen of the IAM user you created at the start of the Storage part of this guide.
+ * **ELASTIC_SEARCH_ARN**: You should find this on the ElasticSearch information screen for your domain. It will be named **Domain ARN**.
+ * **DEV_SERVER_IP**: To allow you to view data in ElasticSearch add your IP address so that you can view ElasticSearch and Kabana without using the IAM login. 
+
+You should now have an ElasticSearch instance that is ready to be used with SAS. 
+
+
+### Access - Elastic Load Balancer
+
+To be able to use ECS and Fargate we need to create an Elastic Load Balancer that will route incoming traffic to the correct ECS Task. To do this navigate to the EC2 service and scroll down to Load Balancers. Click Create load Balancer. 
+
+**Load balancer type**: Application Load Balancer
+
+**Name**: https-LoadBalancer
+
+**Scheme**: Internet facing
+
+**IP address type**: ipv4
+
+**Listeners**: If you want to access your application using http then leave the default listener in place. You should also add a https listener by clicking Add listener and selecting https. 
+
+**Availability Zones**: use the default for VPC and select two zones. 
+
+**Configure Security Settings**: If you have a certificate then assign this to your Load balancer. As load balancers are quite expensive it makes sense to re-use them for different services. If you plan to do this then it is a good idea to create a wildcard certificate and assign it here. 
+
+**Security policy**: BEN WHAT DO YOU SEE HERE?
+
+**Configure Security groups**: BEN WHAT DO YOU SEE HERE?
```

Comment:
Select your default security group. Note you will need to check this default security group has the following inbound ports open 80 and 443. To check this navigate to the following in a new window:

 EC2 > Security Groups > default policy > Inbound Rules